src/Project/ProjectVersionGit.php

```diff
@@ -36,6 +36,16 @@ public function getVersionString()
     }
 
     /**
+     * @return string closest tag of the project, read from git
+     */
+    public function getClosestTagString()
+    {
+        $version = $this->getGitData()['tag'];
+
+        return strstr($version, '-', true);
```

Comment:
`git describe --tags` returns the tag only when on the tagged version. `strstr()` returns `false` when the needle was not found. So it is the wrong function to use, sorry for the tip.
Maybe use `explode('-', $version, 2)[0]` instead.

There is still a problem when a dash "-" is in the tag. Do we care about this?